src/csharp/Microsoft.Spark.AvroTests/Microsoft.Spark.AvroTests.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Why does this require a new project?

Reply:
It needs a new SparkFixture because it needs the spark-avro package with `spark-submit`. I wanted to localize this package to only Avro related tests.

Reply:
Can't you just integrate with the existing E2E test project? I don't see a big benefit of having this as a separate project; note that spark-avro is a part of OSS Spark.

Reply:
I would have to modify [`SparkFixture.cs`](https://github.com/dotnet/spark/blob/main/src/csharp/Microsoft.Spark.E2ETest/SparkFixture.cs) to include the `org.apache.spark:spark-avro` package as mentioned in [documentation here](https://spark.apache.org/docs/latest/sql-data-sources-avro.html#deploying).
If that is ok I can integrate with existing E2E test project.

Reply:
yea that should be ok.